### PR TITLE
add new llm-jp models to llm-engine

### PR DESCRIPTION
**Pull request by ian-scale**
tested by creating a training deployment of the model and successfully sending completions requests through.

### CODE REVIEW

File: model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py [hunk 1]
```diff
@@ -105,6 +105,8 @@
         "code-llama-7b": "codellama/CodeLlama-7b-hf",
         "code-llama-13b": "codellama/CodeLlama-13b-hf",
         "code-llama-34b": "codellama/CodeLlama-34b-hf",
+        "llm-jp-13b-instruct-full-jaster-v1.0": "llm-jp/llm-jp-13b-instruct-full-jaster-v1.0",
```
Comment (yunfeng-scale) [inline model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py:108]: is there a better canonical name? the current names are pretty verbose

Comment (ian-scale) [reply model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py:108]: I agree, it's pretty bad nomenclature but they want to have 2 different llm-jp-13b-instruct models for some reason

Comment (ian-scale) [reply model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py:108]: maybe we could have the model named `llm-jp-13b-instruct-full` and `llm-jp-13b-instruct-full-dolly`?

Comment (yixu34) [reply model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py:108]: Yeah I think those names seem reasonable.

**Review by saiatmakuri — APPROVED** [review]

**Review by yixu34 — APPROVED** [review]

**Pull request merged**